packages/gitgraph-react/rollup.config.js

```diff
@@ -1,3 +1,25 @@
-import build from "../../rollup.config";
+const globals = {
```

Comment:
I kept the original rollup config for this one, as I didn't want to change it. It might be simplified though, we don't need to generate a bundle for browsers for this one IMO.